services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The use of type hints like `list[int]` requires Python 3.9 or later. Ensure your environment is using a compatible version to avoid runtime issues.